src/index.html

```diff
@@ -1,18 +1,76 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
     <title>Product cards</title>
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div
+        alt="APPLE A1419 iMac 27/' Retina 5K Monoblock"
+        class="card__image"
+      ></div>
```

Comment:
Use `img` instead of `div` with  `bacground-image`

Reply:
![image](https://github.com/user-attachments/assets/efecba35-e112-4422-9ba2-e21cb2666fc2)

My bad, i thought i needed to do it with css. Although now it fixed my weird width value